Services/source/util/service_utilities.c

```diff
@@ -141,6 +150,14 @@ void cnv8_D(uint8_t *from, double *to) {
   *to = cnv.cnvD;
 }
 
+uint16_t betole(uint16_t x) {
+#if (SYETEM_ENDIANESS == SYS_BIG_ENDIAN)
+    return (x >> 8) | (num << 8);
```

Comment:
What's num?

Reply:
I don't know why this is here actually. I'll review